app/src/main/res/values/strings.xml

```diff
@@ -417,7 +417,7 @@
     <string name="privacy_policy_url" translatable="false">https://newpipe.net/legal/privacy/</string>
     <string name="read_privacy_policy">Read privacy policy</string>
     <string name="app_license_title">NewPipe\'s License</string>
-    <string name="app_license">NewPipe is copyleft libre software: You can use, study share and improve it at will. Specifically you can redistribute and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.</string>
```

Comment:
This would also need to be changed in the readme and wherever it is also used.

Reply:
I'd keep it like it is

Reply:
It already says "copylefted libre software" in the README.
Yes it could be explained there.